src/CMakeLists.txt

```diff
@@ -646,11 +657,11 @@ add_dependencies(ceph-mon erasure_code_plugins)
 install(TARGETS ceph-mon DESTINATION bin)
 
 # OSD
-if(${HAVE_XFS})
+if(${HAVE_LIBXFS})
```

Comment:
nit, maybe we can be more consistent by just using `if (var)` instead of `if (${var})`. simpler and easier to read. see https://cmake.org/cmake/help/v3.0/command/if.html